utils/node_local_test.py

```diff
@@ -396,8 +396,9 @@ def start(self):
 
             if ready:
                 break
-            if time.time() - start > 10:
+            if time.time() - start > 20:
```

Comment:
Could you please elaborate on why this is needed?
Is the server somehow taking more time to start up than before? Do we have a regression there?

Reply:
This code changed from running on VMs to physical hardware yesterday in https://github.com/daos-stack/daos/commit/326ef68c7300722d166028f458b6b698fffb692b so we expect some timing differences.

I'm not sure why real hardware is slower, but there's more cores and memory/storage to initialise and threads aren't packed as tightly together on cores so that may be an issue.

The other change in this PR is to log the time taken so we can track it, it's not clear from the old logs how long it was taking on VMs but we could look into it.